packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+/// Defines default property values for [PopupMenuEntry]'s [Material].
```

Comment:
It's not just for material since the text style is in here too, should be enough to just say `[PopupMenuEntry] widgets

Reply:
Done